tests/test_commands.py

```diff
@@ -203,3 +203,61 @@ def test_hotline(fake_message, mocker):
     mock_forward_message.assert_called_once_with(
         fake_message.chat.id, fake_message.chat.id, mock_hotline_message_id
     )
+
+
+def test_hbd(fake_message, fake_user, mocker):
+    class FakeBot:
+        def __init__(self):
+            self.user = fake_user
+            self.send_message = MagicMock()
+
+    fake_bot = FakeBot()
+    birthday_boy_or_girl_name = "Tutu Lulul"
+    fake_message.text = '/hbd {}'.format(birthday_boy_or_girl_name)
+    hbd_greetings = "hoi " + birthday_boy_or_girl_name \
+        + " met ultah ya moga sehat dan sukses selalu \xF0\x9F\x8E\x89 \xF0\x9F\x8E\x8A"
+
```

Comment:
Can you format the string like this instead:

```python
hbd_greeting = 'hoi {} met ultah ya blabla'.format(name)
```

Notice that I used `greeting` since it's not a list, and just `name` instead of the lengthy `birthday_boy_or_girl_name`.